ui/src/app/cohort-review/create-review/create-review.component.ts

```diff
@@ -71,9 +68,10 @@ export class CreateReviewComponent implements OnInit {
   createReview() {
     this.creating = true;
     const {ns, wsid, cid} = this.route.snapshot.params;
+    const cdr = this.route.snapshot.data.workspace.cdrVersionId;
```

Comment:
nit - Do we want to call this cdrid? since all other variables that represent an id have it in the name.

Reply:
Done.